### ISSUE TITLE
Testing Metrics

### ISSUE THREAD

**Issue by harshit4084** [zhangjunh/DR-GAN-by-pytorch#12]
The Paper presents some tables on CFP dataset (Table-2) with some metric numbers. How can I test my trained model to get those numbers? 1. As inference 2. While training 

In the Paper metric is defined as: "Results are reported with the average face verification accuracy with standard deviation over 10 folds". I tried to implement it but getting wrong results. Is there any library available for this?

**Comment by zhangjunh** [comment]
@harshit4084 Sorry, I don't konw the answer, and when I finished this code I just evaluate images by my eyes. Maybe you could send an e-mail to the paper's author, and hope you could add your codes to this project after you find it.
